src/lib/components/Legend/Legend.js

```diff
@@ -210,6 +213,7 @@ export class Legend extends React.Component {
     latestTimestamp = latestTimestamp ? <span>Timestamp: {latestTimestamp}</span> : null;
     let primaryLegend;
     let layer;
+    let exportLink = (<a className="glyphicon glyphicon-download-alt" data-download="download" target="_blank" href={layerObj.exportLink} download></a>);
```

Comment:
Should the export link be a component on its own?

Reply:
I thought it would be an overkill since it's a single anchor tag